### ISSUE TITLE
FEATURE REQUEST: alternate mode: STG sidebar not synced with Firefox tab bar

### ISSUE THREAD

**Issue by boldcompany** [Drive4ik/simple-tab-groups#751]
**Is your feature request related to a problem? Please describe.**
Currently, when a user changes STG Groups, Firefox unloads the previous Group of tabs and loads the selected Group of tabs. In some cases, the user may want to have STG act completely separately from Firefox itself - not hiding/showing native Firefox tabs, but simply acting as a separate organizer. After using in this mode, the user may then want to go back to normal STG mode where sidebar and native tab bar are again synced.

Cases where this is useful:
-User has many tabs and groups and wants to quickly organize them while navigating back and forth between groups, but the automatic loading/unloading of tabs by STG with each group change causes unacceptable slowdowns and unnecessary system performance loss. User wants to enter this "organize-only" mode and then go back to regular mode once all tabs are organized.
-User is on a low-performance system and never wants any performance loss when switching STG Groups (due to loading/unloading tabs) but does want a quick way to organize and access tabs. User is willing to accept that native Firefox tab bar will include all Groups' tabs (no hiding) in exchange for the performance boost.

**Describe the solution you'd like**
In this new optional mode (selectable in options), all tabs across all groups would be shown in the native Firefox tab bar at all times. None would be hidden. The STG sidebar would simply act as an organizer for all the open tabs. When a tab is selected in the STG sidebar, the native Firefox tab bar would jump to the selected tab. When a tab is selected in the native Firefox tab bar, the correct STG sidebar group is automatically navigated to and the correct tab autoselected.

If the user utilizes this optional mode and then deselects it, the currently selected Group would be enforced and all tabs outside that group would then be hidden.

**Describe alternatives you've considered**
This is a unique solution for a unique case. No alternative available.

**Additional context**
Please consider: Many users of add-ons like this are using a large number of tabs, which is why they seek an add-on like this in the first place as an organization/usability solution. Options like this feature request are important for users who may be turned away by the laggy performance inherent in switching Groups with hundreds of tabs.

**Comment by Dollique** [comment]
For organizing the Tabs there is the "Manage group" view. You can see all the tabs. You can also set a hotkey for it for faster access. For the purpose of organizing this should be described as an alternative.

**Comment by boldcompany** [comment]
Dollique, thank you so much for adding this note. It is agreed that "manage group" view provides an alternative, and very useful, way of organizing the tabs.

That said, it does not solve the problem of the system load which occurs (and which can be quite heavy on under-resourced systems) which occurs when switching between groups, as STG hides/unhides tabs. The alternative originally suggested here bypasses that by simply leaving all tabs active in the native Firefox tab bar.

**Comment by ygrek** [comment]
Yes please. I would add one more usability point : for me as a new user it was quite surprising/scary to see all my 3K+ tabs disappear when I opened new group. Later I realized they were moved to hidden state and I could uncover them all. Originally I did expect STG to work in the way it is described here - as an independent organizer. Even though I learned my way now, I would still appreciate the proposed mode.

**Comment by grahamperrin** [comment]
> organize

+1 to **Manage groups**

Without switching: 

![2021-02-22 03:04:28](https://user-images.githubusercontent.com/192271/108652120-462d1c80-74bb-11eb-8d30-54190ad97410.png)

Seeking items (a more focused view): 

![image](https://user-images.githubusercontent.com/192271/108652011-f3536500-74ba-11eb-8aee-6dcb16318b26.png)

If the time taken to **initially load** the management view is a concern, then don't close it. Maybe make it a pop-up, for example: 

![image](https://user-images.githubusercontent.com/192271/108652562-5e516b80-74bc-11eb-8394-8cb403adbc18.png)

**Comment by grahamperrin** [comment]
https://github.com/Drive4ik/simple-tab-groups/issues/751#issuecomment-749017937

> … load … when switching between groups, …

You can minimise switching durations by making better use of windows. 

Switching from one window to another should be instantaneous. 

![image](https://user-images.githubusercontent.com/192271/108654807-5ba54500-74c1-11eb-9f0f-7bd01ee94ac0.png)

![image](https://user-images.githubusercontent.com/192271/108654858-74155f80-74c1-11eb-949f-cc81f6aadbb3.png)
